lib/runner/variant-reassignment.js

```diff
@@ -203,10 +225,14 @@ export default class VariantReassignment {
       const product = productSet.values().next().value
```

Comment:
Regarding this condition `else if (productSet.size === 1) {` - what if there is more than just one "conflicting" product in `productSet`? Eg. if there is a product draft with slugs `en: 111, de: 222` and on API there are two products - first with slug `en: 111` and second with `de: 222`. Does it work like when there are more conflicting products then we have to do a reassignment but if there is only one then we will just update it with new values from productDraft?